package.json

```diff
@@ -27,24 +26,26 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "eject": "react-scripts eject",
```

Comment:
rule suggestion: trailing commas would prevent this line from being added to the diff

Reply:
ah this file might be ignored

Reply:
Yep it only lints .tsx, I think thats ok since we won't really be messing with the package.json too much.